### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by tyndria**
Add pure components for login page: navbar with login form and form for registration

screen shot 2017-07-15 at 13 14 41

screen shot 2017-07-15 at 13 19 23

### CODE REVIEW

File: src/sign-up-page/sign-up-page.scss [hunk 1]
```diff
@@ -1,3 +1,22 @@
-.login-container .login-form .control {
-	margin: 0.5rem;
+@import '../index';
```
Comment (Drapegnik) [inline src/sign-up-page/sign-up-page.scss]: you need this for variables?

**Comment by Drapegnik** [comment]
page looks very empty and simple, mb we need something like this https://dansup.github.io/bulma-templates/templates/login.html ?

**Comment by tyndria** [comment]
I thought, it will look better with footer, real navigation bar and logo...

File: src/sign-up-page/sign-up-page.scss (hunk 1 above)
Comment (tyndria) [inline src/sign-up-page/sign-up-page.scss]: Yes

**Comment by Drapegnik** [comment]
[sign-in, sign-up forms](https://trello.com/c/hinxFMLC/9-sign-in-sign-up-forms)

File: src/sign-up-page/sign-up-page.html [hunk 2]
```diff
@@ -8,40 +8,58 @@
...
+									<input type="checkbox"> Я тут першы раз</label></p>
+						</span>
+
+						<a class="social-btn round-btn vk button">
+							<span class="icon"><i class="fa fa-vk"></i></span></a>
+
+						<a class="social-btn round-btn facebook button">
+							<span class="icon"> <i class="fa fa-facebook"></i></span></a>
+
+						<a class="social-btn round-btn twitter button">
```
Comment (Drapegnik) [inline src/sign-up-page/sign-up-page.html]: please fix indentation: same open and close tags should be on one line, or with same indent

File: gulpfile.js [hunk 3]
```diff
@@ -40,7 +40,7 @@ gulp.task('sass:lint', () => gulp.src(`${config.srcPath}/**/*.scss`)
 	}))
 );
 
-gulp.task('sass:watch', () => watch(`${config.srcPath}/**/*.scss`, () => {
+gulp.task('sass:watch', () => watch([`${config.srcPath}/**/*.scss`, `${config.srcPath}/index.sass`], () => {
```
Comment (Drapegnik) [inline gulpfile.js]: Maybe there is a regexp to cover both `. sass` and `.scss`?

Comment (tyndria) [inline gulpfile.js]: Yes, but... Actually, why is this file with sass extension?

Comment (Drapegnik) [inline gulpfile.js]: I have some problems with variables when extension is `.scss`, but you can try fix it)

Comment (tyndria) [inline gulpfile.js]: Ok, I will try

File: gulpfile.js [hunk 4]
```diff
@@ -40,8 +40,9 @@ gulp.task('sass:lint', () => gulp.src(`${config.srcPath}/**/*.scss`)
 	}))
 );
 
-gulp.task('sass:watch', () => watch(`${config.srcPath}/**/*.scss`, () => {
+gulp.task('sass:watch', () => watch([`${config.srcPath}/**/*.scss`, `${config.srcPath}/index.scss`], () => {
```
Comment (Drapegnik) [inline gulpfile.js]: it's unnecessary, `/index.scss` matched by `/**/*.scss` pattern

File: src/index.scss [hunk 5]
```diff
@@ -0,0 +1,34 @@
+// Import the initial variables
+@import '../node_modules/bulma/sass/utilities/initial-variables';
+// Import the rest of Bulma
+@import '../node_modules/bulma/bulma';
```
Comment (Drapegnik) [inline src/index.scss]: you are sure that order is unnecessary? previous version i found in bulma's docs

File: src/sign-up-page/sign-up-page.html [hunk 6]
```diff
@@ -8,40 +8,77 @@
 </head>
 <body>
 
-<div class="login-container">
+<div class="login-main-container">
```
Comment (Drapegnik) [inline src/sign-up-page/sign-up-page.html]: if we call this form `login` maybe it be more clear to name file and directory the same name?

**Comment by Drapegnik** [comment]
screen shot 2017-07-12 at 01 33 29
looks good, I liked it! 👍 

but you still have incorrect indent, and conflicts with master in `sign-up-page.scss`

File: src/sign-up-page/sign-up-page.scss (hunk 1 above)
Comment (Drapegnik) [inline src/sign-up-page/sign-up-page.scss]: hm, after you do this, `bundle.css` contains all libs, and in this case `assets.min.css` is just duplicate it.

File: gulpfile.js (hunk 4 above)
Comment (tyndria) [inline gulpfile.js]: Hm, ok, thanks!

File: src/sign-up-page/sign-up-page.html (hunk 6 above)
Comment (tyndria) [inline src/sign-up-page/sign-up-page.html]: Yes, agree with you, will rename, thanks

File: src/index.scss (hunk 5 above)
Comment (tyndria) [inline src/index.scss]: Will check it

**Comment by Drapegnik** [comment]
@tyndria all is good 👍 you can merge it!

**Pull request merged**